app/scripts/modules/netflix/fastProperties/view/fastPropertyTable.directive.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.netflix.fastProperties.view.tableRow.directive', [
+  require('core/utils/jQuery.js'),
+])
+  .directive('fastPropertyTable', function ($timeout) {
+    return {
+      restrict: 'E',
+      scope: {
+        properties: '=',
+        groupedBy: '=?',
+        showDetails: '&',
+      },
+      link: function (scope, elem) {
+
+        function renderInstances() {
+          var properties = scope.properties;
+
+          let tableHeader = `
+          <table class="table table-hover" style="word-break: break-all">
+          <thead style="border-top: none">
+          <tr>
+            <th width="20%" ng-if="scope.groupedBy !== 'property'">Property</th>
+            <th width="20%">Value</th>
+            <th width="10%" ng-if="scope.groupedBy !== 'app'">Application</th>
+            <th width="5%">Env</th>
+            <th width="10%">Region</th>
+            <th width="10%">Stack</th>
+            <th width="10%">Scope</th>
+          </tr>
+          </thead>
+          <tbody>
+          `;
+
+          let tableFooter = `
+            </tbody>
+          </table>`;
+
+          let innerHtml = tableHeader + properties.map(function(property) {
+
+              return `
+
+              <tr data-property-id="${property.propertyId}">
+                <td>${property.key}</td>
+                <td>${property.value || ''}</td>
+                <td>${property.appId}</td>
+                <td><span class="label label-default account-label account-label-${property.env}">${property.env}</span></td>
+                <td>${property.scope.region}</td>
+                <td>${property.scope.stack}</td>
+                <td>${getBaseOfScope(property.scope)}</td>
+              </tr>
+              `;
```

Comment:
It might not make as big a difference (it's been quite a while, and maybe the browsers have gotten faster, and maybe this is a small enough dataset), but using boring old string concatenation ended up being faster when I had to go down this road for the instance rendering (string templates added 20-30 percent at the time).

If it's sufficiently fast to render right now, I'd leave this as you have it, since it's so much easier to read and maintain.